compiler/noirc_evaluator/src/ssa/ir/function.rs

```diff
@@ -164,7 +164,7 @@ impl Function {
     }
 
     /// Returns the return types of this function.
-    pub(crate) fn returns(&self) -> &[ValueId] {
+    pub(crate) fn returns(&self) -> Vec<ValueId> {
```

Comment:
Can you change this back to returning a `&[ValueId]` so we can avoid the allocation here?

We should be able to return `&[]` in the default case

Reply:
Done